_commands/tools/mpage.md

```diff
@@ -0,0 +1,59 @@
+---
+---
+
+mpage
+---
+
+`mpage` is used to print pages to a postcript printer. It is useful at condensing output, and can condense up to 8 pages onto a single piece of paper.
+
+~~~ bash
+$ mpage [options] <files...>
+~~~
+
+<!--more-->
+
+###Useful Options / Examples
+
+#### `mpage -8 test.txt`
+~~~ bash
+$ls
+test.txt
+$mpage -8 test.txt
+~~~
+
+##### Break it down
+ * The '-8' option sets the condensing from every 8 pages of normal text, to one page printed out. 
```

Comment:
Please surround command line arguments (or any other code) in \`backticks\` to make it appear `like this`.